libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/impl/resolverNamedOptions.kt

```diff
@@ -23,7 +23,11 @@ operator fun MutableMap<String, String>.set(key: DependenciesResolverOptionsName
  */
 enum class DependenciesResolverOptionsName(optionName: String? = null) {
     TRANSITIVE,
-    SCOPE;
+    SCOPE,
+    USERNAME,
+    PASSWORD,
+    PRIVATE_KEY_FILE,
```

Comment:
I don't like the names, taking into account that they will be used (in the lowercased form) in the annotations in the actual scripts. But I don't have much better suggestions either. Maybe `AUTH_KEY_FILE`/`AUTH_KEY_PASSPHRASE` will be a bit better, but I'm not sure. Maybe we should discuss it.

Reply:
I'm ok with the names you suggested, maybe it's even worth getting rid of `AUTH_` prefix

Reply:
Ok, let it be `KEY_FILE`/`KEY_PASSPHRASE`